### PR TITLE
Update to global user groups to remove DataTables

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Update the Groups page (`/groups`) to remove DataTables and use Ant Design.

## Related issue
#413 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [X] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [X] Tests added (or description of how to test) for any new features.
~~* [ ] User documentation updated for UI or technical changes.~~ <-- Update will come in next merge request to do with groups

### CODE REVIEW

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -440,14 +440,27 @@ user.edit.phone.no-numbers=Phone number must contain numbers
...
+
 # Create and edit groups page
 group.create.details=Create Group
 group.name=Group Name
 group.name.required=Required
 group.description=Group Description
 errors.groupname=Oops, there's something wrong with your group name
 group.name.exists=A group with this name already exists, choose a different name.
-group.id=Group ID
+group.id=Group IDE
```
Comment (deepsidhu85) [inline src/main/resources/i18n/messages.properties:463]: Shouldn't this just be `ID` and not `IDE`?

File: src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupsTable.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupsTable.jsx:1]: File comments

File: src/main/webapp/resources/js/pages/UserGroupsPage/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/pages/UserGroupsPage/index.js:1]: File comments

**Comment by deepsidhu85** [comment]
Also on the groups/{group_id} page:

Changing of role does not display a notification to user if it has been successfully updated or not as the following error is thrown in the console

`06 May 2020 16:58:26,756 ERROR ca.corefacility.bioinformatics.irida.ria.web.errors.ExceptionHandlerController:117 - Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; nested exception is java.lang.NumberFormatException: For input string: "undefined"
org.springframework.web.method.annotation.MethodArgumentTypeMismatchException: Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; nested exception is java.lang.NumberFormatException: For input string: "undefined"`

File: src/main/resources/i18n/messages.properties (hunk 1 above)
Comment (joshsadam) [reply src/main/resources/i18n/messages.properties:463]: Oops ;) Fixed in 7e29405c458923968efbe9abd8c0638949d9d403

File: src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupsTable.jsx (hunk 2 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupsTable.jsx:1]: Updated in 3c4bc853f806f3dad3df2644e1d880f41498bf3e

File: src/main/webapp/resources/js/pages/UserGroupsPage/index.js (hunk 3 above)
Comment (joshsadam) [reply src/main/webapp/resources/js/pages/UserGroupsPage/index.js:1]: Updated in 3c4bc853f806f3dad3df2644e1d880f41498bf3e

**Pull request merged**